### PR TITLE
Added describe command to readme

### PR DESCRIPTION
**Pull request by 8ctopus**
First of all thanks for a great tool I just discovered this morning!

While putting it in place in one of my projects, I was wondering how to understand what the rule will do without diving into the code. I found it after some time but it would have been easier had it been in the readme.

This pull request will make it easier for newcomers to find it.

Note: I didn't regenerate the readme.rst because there are lots of other changes besides mine.

### CODE REVIEW

**Comment by 8ctopus** [comment]
sorry I made a mess with the readme, it's my first time here and still learning.

File: src/Console/Command/HelpCommand.php [hunk 1]
```diff
@@ -143,6 +143,15 @@ public static function getHelpCopy()
...
     <info>$ PHP_CS_FIXER_FUTURE_MODE=1 php %command.full_name% -v --diff</info>
 
+Rules
+-----
+
+Use the following command to quickly understand what a rule will do to your code:
+
+.. code-block:: bash
+
+    $ php php-cs-fixer.phar describe align_multiline_comment
```
Comment (julienfalque) [inline src/Console/Command/HelpCommand.php:153]: This line should be wrapped with an `<info>` tag like other code samples.

File: src/Console/Command/HelpCommand.php [hunk 2]
```diff
@@ -143,6 +143,15 @@ public static function getHelpCopy()
 
     <info>$ PHP_CS_FIXER_FUTURE_MODE=1 php %command.full_name% -v --diff</info>
 
+Rules
+-----
+
+Use the following command to quickly understand what a rule will do to your code:
+
+.. code-block:: bash
```
Comment (julienfalque) [inline src/Console/Command/HelpCommand.php:151]: This is added automatically so it doesn't need to be added here (this results in this line being duplicated in the actual README).

File: README.rst [hunk 3]
```diff
@@ -254,6 +254,17 @@ would be default in next MAJOR release (unified differ, estimating, full-width p
...
+Rules
+-----
+
+Use the following command to quickly understand what a rule will do to your code:
+
+.. code-block:: bash
+
+.. code-block:: bash
+
+    $ php php-cs-fixer.phar describe align_multiline_comment
```
Comment (keradus) [inline README.rst:266]: maybe mention the ruleset as well, eg `php-cs-fixer describe @PSR2` ?

**Comment by 8ctopus** [comment]
@julienfalque Thanks for the feedback, I made all requested changes.

File: README.rst (hunk 3 above)
Comment (8ctopus) [reply README.rst]: good idea! I added it.

**Comment by SpacePossum** [comment]
@8ctopus can you help me out and rebase and squash your PR? seems the merge tooling has an issue that stops it from squashing automatically

**Comment by 8ctopus** [comment]
@SpacePossum sorry for the delayed reply, I need to check how to do that without messing the work.

**Comment by SpacePossum** [comment]
no worries and no rush :)
lemme know how it goes, I can do the squashing manually if you prefer that

**Comment by julienfalque** [comment]
Thank you @8ctopus.

**Pull request merged**